Tests/Command/CleanMediaCommandTest.php

```diff
@@ -20,17 +20,6 @@
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Tester\CommandTester;
 use Symfony\Component\DependencyInjection\ContainerInterface;
-use Symfony\Component\Filesystem\Tests\FilesystemTestCase;
```

Comment:
You can keep this line and make this testClass extend directly from it. Also you can remove this line:

`@requires function Symfony\Component\Filesystem\Tests\FilesystemTestCase::setUpBeforeClass`